src/transaction.js

```diff
@@ -403,7 +410,7 @@ Transaction.prototype.request = function(config, callback) {
   );
 
   this.session.request(config, function(err, resp) {
-    if (!self.runFn_ || !err || err.code !== ABORTED) {
+    if (!self.runFn_ || !err || !self.isRetryableErrorCode_(err.code)) {
```

Comment:
We need a test in the `request` describe block that checks that `err.code` is passed to `this.isRetryableErrorCode_`.